### ISSUE TITLE
Url props of pdf preview component is not being used

### ISSUE THREAD

**Issue by Akash044** [VueFilesPreview/vue-files-preview#90]
```
const props = withDefaults(defineProps<PreviewProps>(), {
  url: () => null,
  file: () => null,
})

const fileRender = ref(null)
watch(
    () => props.file,
    (file) => {
      if (file) {
        getFileRenderByFile(file).then(render => (fileRender.value = render)) // fileRender is updated only when we have prop.file
      }
    },
    {immediate: true},
)
</script>

<template>
  <div>
    <iframe class="pdf-iframe" :src="fileRender" frameborder="0"/> // fileRender is null when we don't have prop.file
  </div>
</template>```

**Comment by SmallTeddy** [comment]
yeah, sorry, I forget used it. I will fix it soon!

**Comment by SmallTeddy** [comment]
I fix it in `1.0.37` version
